### ISSUE TITLE
Try to find a way of reading tiles directly from game data

### ISSUE THREAD

**Issue by syntaxaire** [TrashMonks/hagadias#10]
This is desired for several reasons:
* allow the wiki to be updated immediately after a patch with new tiles comes out - especially after wiki updating is made an automatic process driven by patches
* remove the burden of maintaining the modding toolkit zip from Brian and Jason
* decrease the overhead required to set up hagadias, cryptogull and QBE

There may be a Python package capable of reading this data:
https://github.com/HearthSim/UnityPack

**Comment by syntaxaire** [comment]
After looking more deeply into this, I made some progress, but also ran into a wall.

The game assets are stored in `resources.assets` and can be browsed with, for example, Asset Studio (https://github.com/Perfare/AssetStudio/releases). The internal sprite sheets are stored as, for example, `Kobold_DynamicAtlas_Creatures_1`. However, these sprite sheets need a mapping between the image path (e.g. `"Creatures/sw_bearman4.bmp"`) and the X, Y location of the desired tile in the sprite sheet.

In the decompiled game, the part of the code responsible for reading these sprites is in Kobold/SpriteManager.cs/GetTextureInfo. The call that would use the mapping between sprite name and spritesheet location is:
```csharp
exTextureInfo1 = Resources.Load("TextureInfo/" + Path.ToLower().Replace('/', '_').Replace('\\', '_')) as exTextureInfo;```
`Resources.Load` is a call into the Unity library, which implies to me that the sprite sheet coordinates are not determined by the Kobold library, but rather mapped by Unity itself using some secret formula that isn't exposed either in the browsable assets *or* in the Qud code.

In short, the only way to get this information is to be the Caves of Qud executable.

If any of this is wrong or anyone has more advanced knowledge of how this works, I'd love to hear it, but otherwise this might be a dead end.
